roles/contiv_network/files/v2plugin.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description= enable v2plugin after etcd 
```

Comment:
nitpick: extra space between `=` and `enable` here

Reply:
ok. will fix it